scripts/package_building/build_artifacts.sh

```diff
@@ -0,0 +1,399 @@
+#!/bin/bash
+
+set -e
+
+. utils.sh
+
+function install_deps_CentOS {
```

Comment:
install_deps_rhel|install_deps_debian|install_deps_suse

these are our main targets for now.